pages/03-core/standalone-legend.page.tsx

```diff
@@ -0,0 +1,112 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useRef } from "react";
+
+import Button from "@cloudscape-design/components/button";
+import Link from "@cloudscape-design/components/link";
+import SpaceBetween from "@cloudscape-design/components/space-between";
+
+import { CoreLegendProps } from "../../lib/components/core/interfaces";
+import { StandaloneLegend } from "../../lib/components/internal-do-not-use/standalone-legend";
+import { PageSettingsForm, useChartSettings } from "../common/page-settings";
+import { Page } from "../common/templates";
+
+const series: Highcharts.SeriesOptionsType[] = [
+  {
+    name: "A",
+    type: "line",
+    data: [],
+  },
+  {
+    name: "B",
+    type: "line",
+    data: [],
+  },
+  {
+    name: "C",
+    type: "line",
+    data: [],
+  },
+  {
+    name: "X",
+    type: "scatter",
+    data: [{ x: 1601012700000, y: 500000 }],
+    marker: { symbol: "square" },
+    showInLegend: false,
+  },
+];
+
+export default function () {
+  const { chartProps } = useChartSettings();
+  const legendAPI = useRef<CoreLegendProps.LegendAPI | null>(null);
+  const legendProps = {
+    title: chartProps.cartesian.legend?.title,
+    actions: chartProps.cartesian.legend?.actions,
+  } as const;
+  return (
+    <Page
+      title="Standalone legend demo"
```

Comment:
It would be nice to make this page a bit closer to the desired use case.

I would add two charts w/o legend, so that a single legend can control them. I recommend using different chart types, e.g. line and pie - but using the same series.

In that case the buttons to programmatically highlight or toggle items won't be needed - that will be achieved via the charts integration. I would keep the legend title and actions page settings - those are good, and also add the vertical placement option if that is supported for the standalone variant.

Reply:
Very nice page now 👍